frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -52,7 +52,13 @@ export const KnativeOverviewPage: React.ComponentType<
     if (kindsInFlight) {
       return !knativeModels ? null : <LoadingBox />;
     }
-    const resourceModel = knativeModels.find((model) => model.kind === item.obj.kind);
+    const apiInfo = groupVersionFor(item.obj.apiVersion);
+    const resourceModel = knativeModels.find(
+      (model) =>
+        model.kind === item.obj.kind &&
+        apiInfo.group === model.apiGroup &&
+        apiInfo.version === model.apiVersion,
```

Comment:
heh, flipped the check to the other side of the equals :) Not a big issue, just odd to look at.